openmc/data/data.py

```diff
@@ -208,6 +211,110 @@ def atomic_weight(element):
     return None if weight == 0. else weight
 
 
+def water_density(temperature, pressure=0.1013):
+    """Return the density of liquid water at a given temperature and pressure.
+
+    The density is calculated from a polynomial fit using equations and values
+    from the 2012 version of the IAPWS-IF97 formulation.  Only the equations
+    for region 1 are implemented here.
```

Comment:
Suggest tieing "region 1" more directly with the next paragraph (i.e., the P/T limits et al define region 1)